### ISSUE TITLE
Module not found: Can't resolve 'useBoolean'

### ISSUE THREAD

**Issue by nghiepit** [kitze/react-hanger#63]
Hey,

Just upgrade to `2.4.0` and error compile:

```
error - ./node_modules/react-hanger/esm/useDocumentReady.js:1:0
Module not found: Can't resolve 'useBoolean'
```

**Comment by RIP21** [comment]
Jeeeez. Have no idea how this is not caught on the build phase. WIll hotfix in a miinute.

**Comment by RIP21** [comment]
Fixed in 2.4.1. I hope.
@nghiepit thanks for the report. Please check :)

**Comment by nghiepit** [comment]
tks @RIP21 
It works again
